package.json

```diff
@@ -22,8 +22,7 @@
   "license": "MIT",
   "dependencies": {
     "jsonml.js": "^0.1.0",
-    "lodash.camelcase": "^4.1.1",
-    "lodash.isfunction": "^3.0.8"
+    "lodash": "^4.17.4"
```

Comment:
probably fine, though we could just specify `^4` to allow maximum deduping downstream?

Reply:
Ok, i've done a search, and the lowest common version is `4.14.1`

https://github.com/search?l=JSON&p=2&q=org%3ACondeNast+jsonmltoreact&type=Code

- [autopilot-ad](https://github.com/CondeNast/autopilot-ad/blob/a2434fdb29f5e72356214eb511f2adc3b0a038d2/package.json)
- [autopilot-teen-vogue](https://github.com/CondeNast/autopilot-teen-vogue/blob/527ccb72c7ace773c897bd0961ec5af1281cbc89/package.json)
- [autopilot-self](https://github.com/CondeNast/autopilot-self/blob/86894083ba950c995979b11e7755b6833f55343a/package.json)

cc @dcolucci